tests/test_hooks.py

```diff
@@ -135,3 +135,17 @@ def pytest_bdd_after_scenario(request, feature, scenario):
 
     assert scenario_1.name == "Scenario 1"
     assert scenario_2.name == "Scenario 2"
+
+
+def test_pytest_unconfigure_without_configure(pytester):
+    pytester.makeconftest(
```

Comment:
please add a docstring explaining that this can happen in case plugins cause pytest to exit early

Reply:
Added docstring